pkgs/applications/science/electronics/kicad/base.nix

```diff
@@ -56,9 +56,11 @@ stdenv.mkDerivation rec {
   # tagged releases don't have "unknown"
   # kicad nightlies use git describe --dirty
   # nix removes .git, so its approximated here
+  # "-1" appended to indicate we're adding a patch
   postPatch = ''
     substituteInPlace CMakeModules/KiCadVersion.cmake \
-      --replace "unknown" ${builtins.substring 0 10 src.rev}
+      --replace "unknown" "${builtins.substring 0 10 src.rev}-1" \
+      --replace "${version}" "${version}-1"
```

Comment:
We don't use patchlevels in nixpkgs because the build process of a package can be amended in many different ways, and all those changes can be tracked by looking at the store paths hashes and inside drv files. Patchlevels can't keep up with they, they will quickly become outdated.

Reply:
my goal with this was to make it clear in a kicad bug report (which contains that string) which commit the package was based on, while still making it clear a patch was added